### PR TITLE
feat(cli): export metric physical tables first

### PR DESCRIPTION
**Pull request by WenyXu**
I hereby agree to the terms of the [GreptimeDB CLA](https://github.com/GreptimeTeam/.github/blob/main/CLA.md).

## What's changed and what's your intention?

Export metric physical tables first

## Checklist

- [ ]  I have written the necessary rustdoc comments.
- [ ]  I have added the necessary unit tests and integration tests.
- [x]  This PR does not require documentation updates.

### CODE REVIEW

File: src/cmd/src/cli/export.rs [hunk 1]
```diff
@@ -270,7 +300,7 @@ impl Export {
             })
             .count();
 
-        info!("success {success}/{db_count} jobs");
+        info!("Success {success}/{db_count} jobs");
```
Comment (killme2008) [inline src/cmd/src/cli/export.rs:303]: 👍  Can we add some timing info to the final logs? Such as `Success {success}/{db_count} jobs, Cost {cost} seconds.` etc.

File: src/cmd/src/cli/export.rs [hunk 2]
```diff
@@ -202,10 +225,17 @@ impl Export {
...
                 t.push(value);
             }
-            result.push((t[0].clone(), t[1].clone(), t[2].clone()));
+            let table = (t[0].clone(), t[1].clone(), t[2].clone());
+            // Ignores the physical table
+            if !metric_physical_tables.contains(&table) {
+                result.push(table);
+            }
         }
+        let mut tables = Vec::with_capacity(metric_physical_tables.len() + result.len());
```
Comment (killme2008) [inline src/cmd/src/cli/export.rs:234]: Looks like we don't need to create a new Vector, just extend the `metric_physical_tables.extend(result)` and return it.

And it's better to rename the `result` to a meaningful name.

Comment (WenyXu) [reply src/cmd/src/cli/export.rs:234]: The `metric_physical_tables` is a `HashSet`

**Review by fengjiachun — APPROVED** [review]
LGTM

**Review by waynexia — APPROVED** [review]

**Pull request merged**